webboot/webboot.go

```diff
@@ -153,7 +148,16 @@ func main() {
 		log.Fatal(err)
 	}
 
-	err = write(read, filename)
+	file, err := os.OpenFile("/dev/pmem0", os.O_APPEND|os.O_WRONLY, 0600)
+	if err != nil {
+		log.Fatal(err)
+	}
+
```

Comment:
just put an io.Copy(file, read)
here
also change the name of 
file, err := os.OpenFile("/dev/pmem0", os.O_APPEND|os.O_WRONLY, 0600)
pmem, err := os.OpenFile("/dev/pmem0", os.O_APPEND|os.O_WRONLY, 0600)

and change the name read to something a little nicer, like, iso, maybe?

Reply:
Okay, I will change the variable "file" to "pmem," and "read" to a variable name such as "iso," so it is very clear to the audience.